src/components/shared/Buttons/Button.jsx

```diff
@@ -9,13 +9,15 @@ const Button = ({
   onClicked,
   children,
   isDisabled,
+  dataTestId // added to ease testing of this component
```

Comment:
It is wrong to pass this as props. It is available as an HTML API. Just define it on the element you want.

Reply:
Due to the way the component was defined if we pass it to `button` directly, It means every component that uses this component will also inherit the static `data-testid` prop in this button and so test click simulations will affect all the buttons. I was trying to avoid that, that's why I passed it this way

Reply:
Every test is scoped within its own function. This should not happen.

Reply:
Yes, that's true. I just remembered that there's always "cleanup" after each test runs. Thanks